docs/src/config.ts

```diff
@@ -0,0 +1,74 @@
+export const SITE = {
+  title: 'Media Chrome Docs',
+  description: 'Your website description.',
+  defaultLanguage: 'en_US',
+};
+
+export const OPEN_GRAPH = {
+  image: {
+    src: 'https://www.media-chrome.org/favicon/android-icon-48x48.png',
+    alt: 'closing html tag on a pink background',
+  },
+  twitter: 'MuxHQ',
+};
+
+// This is the type of the frontmatter you put in the docs markdown files.
+export type Frontmatter = {
+  title: string;
+  description: string;
+  layout: string;
+  image?: { src: string; alt: string };
+  dir?: 'ltr' | 'rtl';
+  ogLocale?: string;
+  lang?: string;
+};
+
+export const KNOWN_LANGUAGES = {
+  English: 'en',
+} as const;
+export const KNOWN_LANGUAGE_CODES = Object.values(KNOWN_LANGUAGES);
+
+export const GITHUB_EDIT_URL = `https://github.com/muxinc/media-chrome/tree/main/docs`;
+
+export const COMMUNITY_INVITE_URL = `https://github.com/muxinc/media-chrome/discussions`;
+
+// See "Algolia" section of the README for more information.
+export const ALGOLIA = {
+  indexName: 'dev_media_chrome',
+  appId: '57LJZR6XKN',
+  apiKey: 'ec6de967b91bcd3b0ea96b5fcfedaa7d',
+};
```

Comment:
Is this hooked up? Not getting any results. Maybe it needs to be more officially deployed?